client/state/ui/guided-tours/contexts.js

```diff
@@ -7,65 +7,132 @@ import {
 	getSectionName,
 	getSelectedSite,
 	getSelectedSiteId,
-	isPreviewShowing as isPreviewShowingSelector,
 } from 'state/ui/selectors';
 import { getLastAction } from 'state/ui/action-log/selectors';
 import { getCurrentUser, canCurrentUser } from 'state/current-user/selectors';
 import { hasDefaultSiteTitle } from 'state/sites/selectors';
 
+const WEEK_IN_MILLISECONDS = 7 * 1000 * 3600 * 24;
+
+/*
+ * Returns a selector that tests if the current user is in a given section
+ *
+ * @param {String} sectionName Name of section
+ * @return {Function} Selector function
+ */
 export const inSection = sectionName => state =>
 	getSectionName( state ) === sectionName;
 
+/*
+ * Returns a selector that tests if a feature is enabled in config
+ *
+ * @param {String} feature Name of feature
+ * @return {Function} Selector function
+ */
 export const isEnabled = feature => () =>
 	config.isEnabled( feature );
 
-export const isPreviewNotShowing = state =>
-	! isPreviewShowingSelector( state );
-
-export const isPreviewShowing = state =>
-	isPreviewShowingSelector( state );
-
+/*
+ * Returns milliseconds since registration date of the current user
+ *
+ * @param {Object} state Global state tree
+ * @return {Number|Boolean} Milliseconds since registration, false if cannot be determined
+ */
 const timeSinceUserRegistration = state => {
 	const user = getCurrentUser( state );
 	const registrationDate = user && Date.parse( user.date );
 	return registrationDate ? ( Date.now() - registrationDate ) : false;
 };
 
-const WEEK_IN_MILLISECONDS = 7 * 1000 * 3600 * 24;
+/*
+ * Returns true if the user is considered "new" (less than a week since registration)
+ *
+ * @param {Object} state Global state tree
+ * @return {Boolean} True if user is new, false otherwise
+ */
 export const isNewUser = state => {
 	const userAge = timeSinceUserRegistration( state );
 	return userAge !== false ? userAge <= WEEK_IN_MILLISECONDS : false;
 };
 
+/*
+ * Returns a selector that tests if the user is older than a given time
+ *
+ * @param {Number} age Number of milliseconds
+ * @return {Function} Selector function
+ */
 export const isUserOlderThan = age => state => {
 	const userAge = timeSinceUserRegistration( state );
 	return userAge !== false ? userAge >= age : false;
 };
 
+/*
+ * Returns a selector that tests if the user has registered before given date
+ *
+ * @param {Date} date Date of registration
+ * @return {Function} Selector function
+ */
 export const hasUserRegisteredBefore = date => state => {
 	const compareDate = date && Date.parse( date );
 	const user = getCurrentUser( state );
 	const registrationDate = user && Date.parse( user.date );
 	return ( registrationDate < compareDate );
 };
 
+/*
+ * Returns a selector that tests whether the user has interacted with a given component
```

Comment:
Can we add something like "See also the [TrackInteractions component](client/components/track-interactions/)."?